client/me/purchases/manage-purchase/index.jsx

```diff
@@ -230,7 +230,7 @@ class ManagePurchase extends Component {
 		const { isAtomicSite, purchase, translate } = this.props;
 		const { id } = purchase;
 
-		if ( ! isCancelable( purchase ) || isPartnerPurchase( purchase ) || ! this.props.site ) {
+		if ( ! isCancelable( purchase ) || isPartnerPurchase( purchase ) || ! this.props.siteSlug ) {
```

Comment:
Note sure if the `this.props.siteSlug` check is required here, since that's already supposed to be a required property.  I guess it doesn't hurt though :smile:

Reply:
Oh, good catch. I guess I'm not sure what's better here: leaving it out and just letting propTypes warn us when it fails, or keeping it in, even though it shouldn't ever be encountered if propTypes is preventing us from introducing those edge cases. Since I put it in during this PR, I'll just take it out, unless the extra defensiveness is encouraged :)